Stormshield/stormshield-endpoint-security/_meta/fields.yml

```
@@ -0,0 +1,24 @@
+stormshield.additionaldata.incidentguid:
+  type: keyword
+  description: stormshield incident guid
+  name: stormshield.additionaldata.incidentguid
+
+stormshield.specificData.targetProcess.processcommandline:
+  type: keyword
+  description: stormshield process command line
+  name: stormshield.specificData.targetProcess.processcommandline
```

I suggest using `stormshield.ses` as prefix and `process.target.command_line`  for the name of the :
```suggestion
stormshield.ses.process.target.command_line:
  type: keyword
  description: stormshield process command line
  name: stormshield.ses.process.target.command_line
```